### PR TITLE
[RNMobile] Added support for giphy and pexels images

### PR DESCRIPTION
**Pull request by marecar3**
## Description
Added support for giphy and pexels images

## How has this been tested?

Follow instructions on gb-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1469

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

**Review by Tug — APPROVED** [review]
Tested on WPAndroid. LGTM 👍 

(`extra` could have been a better naming than `other` but I feel I'm nitpicking here)

File: packages/block-library/src/media-text/media-container.native.js [hunk 1]
```diff
@@ -164,14 +164,16 @@ class MediaContainer extends Component {
 		const { finalWidth, finalHeight, imageWidthWithinContainer, isUploadFailed, retryMessage } = params;
 		const opacity = isUploadInProgress ? 0.3 : 1;
 
+		const conttentStyle = ! imageWidthWithinContainer ? styles.content : styles.contentCentered;
```
Comment (cameronvoell) [inline packages/block-library/src/media-text/media-container.native.js]: extra t: "conttentStyle" => "contentStyle"

File: packages/block-library/src/media-text/media-container.native.js [hunk 2]
```diff
@@ -164,14 +164,16 @@ class MediaContainer extends Component {
...
+
 		return (
 			<TouchableWithoutFeedback
 				accessible={ ! isSelected }
 				onPress={ this.onMediaPressed }
 				onLongPress={ openMediaOptions }
 				disabled={ ! isSelected }
 			>
-				<View style={ styles.content }>
+				<View style={ conttentStyle }>
```
Comment (cameronvoell) [inline packages/block-library/src/media-text/media-container.native.js]: See above "conttentStyle"

**Review by cameronvoell — APPROVED** [review]
Looks good to me, just the small typo with the variable name I commented on. 👍

**Comment by marecar3** [comment]
> (extra could have been a better naming than other but I feel I'm nitpicking here)
The idea for the next iteration of this would be to send all options from native app to JS side (we have already opened the issue for that) so for sure we will change the prefix.

File: packages/block-library/src/media-text/media-container.native.js (hunk 1 above)
Comment (marecar3) [reply packages/block-library/src/media-text/media-container.native.js]: Tnx! Fixed!

File: packages/block-library/src/media-text/media-container.native.js (hunk 2 above)
Comment (marecar3) [reply packages/block-library/src/media-text/media-container.native.js]: Tnx! Fixed!

**Pull request merged**